ui/app/components/clients/horizontal-bar-chart.js

```diff
@@ -162,15 +162,10 @@ export default class HorizontalBarChart extends Component {
       .style('opacity', '0')
       .style('mix-blend-mode', 'multiply');
 
-    const actionBarSelection = chartSvg.selectAll('rect.action-bar');
-
-    const compareAttributes = (elementA, elementB, attr) =>
-      select(elementA).attr(`${attr}`) === select(elementB).attr(`${attr}`);
-
     // MOUSE EVENTS FOR DATA BARS
     actionBars
-      .on('mouseover', (data) => {
-        const hoveredElement = actionBars.filter((bar) => bar[labelKey] === data[labelKey]).node();
+      .on('mouseover', (event, data) => {
```

Comment:
I suppose how d3 handles events is what changed? I tried to add test coverage but it was still too flaky, but when this chart is updated to use `Lineal` there will be adequate coverage!